<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserGUI</class>
 <widget class="QWidget" name="UserGUI">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserGUI</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Breed:</string>
         </property>
         <property name="buddy">
          <cstring>breedLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="breedLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Name:</string>
         </property>
         <property name="buddy">
          <cstring>nameLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="nameLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Age:</string>
         </property>
         <property name="buddy">
          <cstring>ageLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="ageLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Photo:</string>
         </property>
         <property name="buddy">
          <cstring>linkLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="linkLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="adoptDogButton">
         <property name="text">
          <string>Adopt the dog</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="seePhotoButton">
         <property name="text">
          <string>See photo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextButton">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="seeAllDogsButton">
       <property name="text">
        <string>See all dogs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="seeSpecificDogsButton">
       <property name="text">
        <string>See dogs of specific breed and age:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Breed:</string>
         </property>
         <property name="buddy">
          <cstring>specificBreedLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="specificBreedLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Age:</string>
         </property>
         <property name="buddy">
          <cstring>specificAgeLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="specificAgeLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="seeAdoptionListButton">
       <property name="text">
        <string>See the adoption list</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>seeAllDogsButton</sender>
   <signal>clicked()</signal>
   <receiver>UserGUI</receiver>
   <slot>seeAllDogs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked()</signal>
   <receiver>UserGUI</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adoptDogButton</sender>
   <signal>clicked()</signal>
   <receiver>UserGUI</receiver>
   <slot>adoptDog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>seeSpecificDogsButton</sender>
   <signal>clicked()</signal>
   <receiver>UserGUI</receiver>
   <slot>seeDogsByBreedAndAge()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>seeAdoptionListButton</sender>
   <signal>clicked()</signal>
   <receiver>UserGUI</receiver>
   <slot>seeAdoptionList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>seePhotoButton</sender>
   <signal>clicked()</signal>
   <receiver>UserGUI</receiver>
   <slot>seePhoto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>seeAllDogs()</slot>
  <slot>next()</slot>
  <slot>adoptDog()</slot>
  <slot>seeDogsByBreedAndAge()</slot>
  <slot>seeAdoptionList()</slot>
  <slot>seePhoto()</slot>
 </slots>
</ui>
